draft-ietf-mls-protocol.md

```diff
@@ -1243,18 +1243,24 @@ messages refer to KeyPackages for the members being welcomed, and Commits refer
 to Proposals they cover.  These identifiers are computed as follows:
 
 ~~~ tls
-opaque HashReference[16];
+opaque HashReference[Hash.length];
```

Comment:
```suggestion
opaque HashReference<V>;
```

Elsewhere, where we have values whose size depends on the ciphersuite, we use vectors instead of arrays.  See, e.g., HPKEPubicKey.  It costs a byte, but it means the parsing/serialization logic doesn't have to know about ciphersuites.